src/dashboard/apps/consent/templates/consent/includes/_consent_summary_upcoming_card_content.html

```diff
@@ -0,0 +1,62 @@
+{% load i18n %}
+
+{% if entities %}
+  <strong class="fr-mt-2w">Autoriser la DGEC à suivre vos unités d’exploitation par anticipation</strong>
+
+  <div class="fr-table fr-table--no-caption fr-table--bordered">
+    <div class="fr-table__wrapper">
+      <div class="fr-table__container">
+        <div class="fr-table__content">
+
+          <table>
+            <caption>Autoriser la DGEC à suivre vos unités d’exploitation par anticipation</caption>
+            <thead>
+              <tr>
+                <th>
+                  <div class="fr-cell__title">Entreprise</div>
+                </th>
+                <th>
+                  <div class="fr-cell__title fr-cell--center">
+                    Nombre de points de livraison en attente
+                  </div>
+                </th>
+                <th>
+                  <div class="fr-cell__title fr-cell--center">
+                    Action
+                  </div>
+                </th>
+              </tr>
+            </thead>
+            <tbody>
+              {% for entity in entities %}
+                {% if entity.count_upcoming_consents %}
+                <tr id="undefined-row-key-{{ forloop.counter }}" data-row-key="forloop.counter">
+                  <td>
+                    <p>{{ entity.name }}</p>
+                  </td>
+                  <td class="fr-cell--center">
+                    <p class="fr-badge fr-badge--new">
+                      {% with consent_total=entity.get_upcoming_consents.count entity_count=entity.count_upcoming_consents pluralize=entity.count_upcoming_consents|pluralize %}
+                      {{ entity_count }} PDL en attente sur {{ consent_total }}
```

Comment:
```suggestion
                      {{ entity_count }} PDL(s) en attente sur {{ consent_total }}
```

or handle the plural form using the appropriate template tag?